docs/release-notes/release-0-9-1.md

```diff
@@ -0,0 +1,107 @@
+# v0.9.1
+
+Release date: August 02, 2024
+
```

Comment:
```suggestion
## 👍 Highlights
```

Reply:
> 

It's not actually *highlights*. I folllowed the style here https://github.com/GreptimeTeam/docs/blob/2603ec9de98073357f2a134e0cd9849e18f17236/docs/release-notes/release-0-8-1.md?plain=1#L5-L7